tuist/tuist
# Remove broken package URL validation

Resolves https://github.com/tuist/tuist/issues/YYY

### Short description 📝

The new URL API should be used by swift version and not OS.

### How to test the changes locally 🧐

Using the if #available(macOS 14.0, *) is not exhaustive in many cases like running macos 13 (ventura) with xcode 15. 
We should check the available Swift version which expose this new API as part of the Foundation framework.

I have run into this issue when building advanced versions (3.28.0 and above) of Tuist for Homebrew

> Include a set of steps for the reviewer to test the changes locally (see [the documentation](https://docs.tuist.io/contributors/testing-strategy) for reference).

### Contributor checklist ✅


### Reviewer checklist ✅
